CP01CNPM/CP01CNPM_R00002_ingest.csv

```diff
@@ -0,0 +1,11 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/cg_data/stc1/imm/adcpt/adcpt*.DAT,CP01CNPM-RII01-02-ADCPTG010,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/ADCP_sn*/*.000,CP01CNPM-RII01-02-ADCPTG010,recovered_inst,Available,
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/cg_data/stc1/irid2shore/stc_status.*_*.txt,CP01CNPM-SBS01-00-STCENG000,recovered_host,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/cg_data/stc1/3dmgx3/*_*.3dmgx3.log,CP01CNPM-SBS01-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/CWFP_sn*/*/E*.DA?,CP01CNPM-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/CWFP_sn*/*/A*.DA?,CP01CNPM-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/CWFP_sn*/*/C*.DA?,CP01CNPM-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/CWFP_sn*/*/C*.DA?,CP01CNPM-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/CWFP_sn*/*/E*.DA?,CP01CNPM-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/CWFP_sn*/*/E*.DA?,CP01CNPM-WFP01-05-PARADK000,recovered_wfp,Available,
```

Comment:
All the WFP lines have ".DA?" instead of ".DAT".

Reply:
Using ".DAT?" allows the filemask to account for cases when there are .DAC files. I think those file types occur when a file doesn't complete transferring, but I could be wrong. @johnmlund